crates/node/src/service.rs

```diff
@@ -196,7 +198,7 @@ where
         keystore_container,
         select_chain,
         transaction_pool,
-        other: (block_import, grandpa_link, telemetry, madara_backend),
+        other: (block_import, grandpa_link, telemetry, Arc::clone(deoxys_backend)),
```

Comment:
why using an arc here

Reply:
This is because the `deoxys_backend` returned when calling `open` is an `&Arc`. This ensure no direct access to the underlying singleton is possible by forcing the `Arc` to be cloned, should it be passed to another function.